extension-api/app/api/routes.py

```
@@ -182,144 +227,178 @@ async def get_top_collections(request: Request, authorization: Optional[str] = H
+        top_collection_ids = (
+            user_result["top_collections"] if user_result["top_collections"] else []
+        )
+
         if not top_collection_ids:
             return TopCollectionsResponse(data=[])
-        
+
         # Convert list to format for SQL IN clause
-        placeholders = ','.join(['%s'] * len(top_collection_ids))
-        
+        placeholders = ",".join(["%s"] * len(top_collection_ids))
```

Ruff (0.13.3)

Local variable `placeholders` is assigned to but never used

Remove assignment to unused variable `placeholders`

(F841)